package.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "@tanstack/react-query": "^5.59.17",
-    "starknetkit": "^2.3.0"
```

Comment:
remove it from root folder, it should be installed to `frontend` (please make sure you are using yarn, not npm)